multiqc/modules/anglerfish/anglerfish.py

```diff
@@ -0,0 +1,247 @@
+        try:
+            for k in parsed_json["undetermined"]:
+                if len(k) > 0:
+                    self.anglerfish_data[s_name]["undetermined_count_{}".format(index)] = float(k["count"])
+                    total_count += float(k["count"])
+                    self.anglerfish_data[s_name]["undetermined_index_{}".format(index)] = k["undetermined_index"]
+                    index += 1
+            self.anglerfish_data[s_name]["undetermined_amount"] = index
+        except (KeyError):
+            # No undetermined in file or undetermined missing info
+            self.anglerfish_data[s_name]["undetermined_amount"] = -1
+        self.anglerfish_data[s_name]["total_count"] = total_count
+
+        # Save total amount of input reads
+        self.anglerfish_data[s_name]["total_read"] = float(parsed_json["paf_stats"][0]["input_reads"][0])
+
+    # General stats table
+    def anglerfish_general_stats_table(self):
+        """Add Anglerfish statistics to the general statistics table"""
+        # Prepp data for general stat table
+        ## Multiple sample names per file requires dict where the first key is not file name
+        data = {}
+        for s_name in self.anglerfish_data:
+            total_read = self.anglerfish_data[s_name]["total_read"]
+            total_count = self.anglerfish_data[s_name]["total_count"]
+            try:
+                for k in range(self.anglerfish_data[s_name]["sample_stats_amount"]):
+                    key = self.anglerfish_data[s_name]["sample_name_{}".format(k)]
+                    data[key] = {}
+                    data["undetermined"] = {}
+                    reads = self.anglerfish_data[s_name]["#reads_{}".format(k)]
+                    data[key]["#reads"] = reads
+                    data[key]["mean_read_len"] = self.anglerfish_data[s_name]["mean_read_len_{}".format(k)]
+                    data[key]["std_read_len"] = self.anglerfish_data[s_name]["std_read_len_{}".format(k)]
+                    data[key]["library"] = float((reads / total_read) * 100)
+                    data["undetermined"]["library"] = float((total_count / total_read) * 100)
+            except (KeyError):
+                log.debug("No general stats table generated from Anglerfish json: {}".format(s_name))
+            except (ZeroDivisionError):
+                log.debug("No library in general stats table generated from Anglerfish json: {}".format(s_name))
+
+        headers = OrderedDict()
+        headers["library"] = {
+            "namespace": "Anglerfish",
+            "title": "library",
+            "description": "Amount of library used. Calculated from a samples #reads divided by total number of input reads",
+            "max": 100,
+            "min": 0,
+            "scale": "PuBu-rev",
+            "suffix": " %",
+        }
+
+        headers["#reads"] = {
+            "namespace": "Anglerfish",
+            "title": "#reads",
+            "description": "Amount of reads",
```

Comment:
I don't love the word "Amount" used in this context.

```suggestion
            "description": "Total number of reads",
```